packages/orbit-components/src/Icon/Icon.stories.tsx

```diff
@@ -1,129 +1,98 @@
+
+const meta: Meta<IconPropsAndCustomArgs> = {
+  title: "Icon",
+  component: OrbitIcon,
 
-Default.story = {
-  name: "Default",
   parameters: {
     info: "We use icons to draw attention to specific actions in our products. Visit Orbit.Kiwi for more detailed guidelines.",
   },
-};
 
-Default.args = {
-  size: ICON_SIZES.MEDIUM,
-  color: ICON_COLORS.PRIMARY,
-  source: "Airplane",
-  dataTest: "test",
-  ariaLabel: "label",
-  ariaHidden: true,
-};
+  args: {
+    size: ICON_SIZES.MEDIUM,
+    color: ICON_COLORS.PRIMARY,
+    source: "Airplane",
+    ariaLabel: "label",
+    ariaHidden: true,
+    customColor: "#ABCDEF",
+  },
 
-Default.argTypes = {
-  size: {
-    options: Object.values(ICON_SIZES),
-    control: {
-      type: "select",
+  argTypes: {
+    size: {
+      options: Object.values(ICON_SIZES),
+      control: {
+        type: "select",
+      },
     },
-  },
-  color: {
-    options: Object.values(ICON_COLORS),
-    control: {
-      type: "select",
+    color: {
+      options: Object.values(ICON_COLORS),
+      control: {
+        type: "select",
+      },
     },
-  },
-  source: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
+    source: {
+      options: Object.keys(Icons),
+      control: {
+        type: "select",
+      },
     },
   },
 };
 
-export const CustomColor = ({ size, customColor, source }) => {
-  const Icon = Icons[source];
+export default meta;
+type Story = StoryObj<IconPropsAndCustomArgs>;
 
-  return <Icon size={size} customColor={customColor} />;
-};
-
-CustomColor.story = {
-  name: "Custom Color",
-  parameters: {
-    info: "We use icons to draw attention to specific actions in our products. Visit Orbit.Kiwi for more detailed guidelines.",
+export const Default: Story = {
+  render: ({ source, ...args }) => {
+    const Icon = getIcon(source);
+    return <Icon {...args} />;
   },
 };
 
-CustomColor.args = {
-  size: ICON_SIZES.MEDIUM,
-  customColor: "#ABCDEF",
-  source: "Airplane",
-};
+export const CustomColor: Story = {
+  render: ({ source, ...args }) => {
+    const Icon = getIcon(source);
 
-CustomColor.argTypes = {
-  size: {
-    options: Object.values(ICON_SIZES),
-    control: {
-      type: "select",
-    },
+    return <Icon {...args} />;
   },
-  source: {
-    options: Object.keys(Icons),
-    control: {
-      type: "select",
-    },
+
+  args: {
+    customColor: "#ff0000",
   },
 };
 
-export const ReversedOnRtl = ({ source }) => {
-  const Icon = Icons[source];
-  return (
-    <RenderInRtl>
-      <Icon reverseOnRtl />
-    </RenderInRtl>
-  );
-};
+export const ReversedOnRtl: Story = {
+  render: ({ source, ...args }) => {
+    const Icon = getIcon(source);
 
-ReversedOnRtl.story = {
-  name: "Reversed on RTL",
-  parameters: {
-    info: "We use icons to draw attention to specific actions in our products. Visit Orbit.Kiwi for more detailed guidelines.",
+    return (
+      <RenderInRtl>
+        <Icon reverseOnRtl {...args} />
```

Comment:
You added args here but you disabled controls, is that on purpose? I think at least the icon could be in controls, no? 🤔